CHANGES.rst

```diff
@@ -1,6 +1,11 @@
 30-cvs (in development)
 =======================
 
+- Improvements:
```

Comment:
Please list this as `**Breaking change**:`, as it'll break configurations that try to set this variable explicitly.